.c8rc.json

```diff
@@ -7,5 +7,5 @@
   "statements": 60,
   "src": ["src"],
   "all": true,
-  "reporter": ["text", "text-summary"]
+  "reporter": ["text", "text-summary", "lcov"]
```

Comment:
worked as expected,

[without `lcov` reporter](https://app.codecov.io/github/jnosornov/advent-of-code/commit/ea20694ba9daa66a8a3e4031cf13a2e75b2410df)

<img width="1337" alt="Screenshot 2023-07-02 at 9 43 27 PM" src="https://github.com/jnosornov/advent-of-code/assets/31334379/aa529cae-da34-4714-a86e-fe01e1613f1d">

[with `lcov` reporter](https://app.codecov.io/github/jnosornov/advent-of-code/commit/74dbaa432249e7ced0f789fc54a7accf7c18ea4f/tree)

<img width="1333" alt="Screenshot 2023-07-02 at 9 45 13 PM" src="https://github.com/jnosornov/advent-of-code/assets/31334379/97afc8bc-2f73-4517-aa46-f069a9cf6470">